ceph/object-storage/nano/base/persistentvolumeclaim.yaml

```diff
@@ -0,0 +1,35 @@
+---
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: cn-pv-claim-var
+  labels:
+    app: ceph
+    daemon: nano
+spec:
+  # Read more about access modes here: http://kubernetes.io/docs/user-guide/persistent-volumes/#access-modes
+  accessModes:
+  - ReadWriteOnce
+  resources:
+    requests:
+        storage: 10Gi
+  # Uncomment and add storageClass specific to your requirements below. Read more https://kubernetes.io/docs/concepts/storage/persistent-volumes/#class-1
+  #storageClassName:
+---
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: cn-pv-claim-etc
+  namespace: $(namespace)
```

Comment:
I believe namespace is added by default - you don't need to explicitely add the variable

Reply:
Updated.